### PR TITLE
feat: add docs-links-check ci and fix dead links

### PR DESCRIPTION
**Pull request by Selflocking**
Fix: https://github.com/casbin/casbin-website-v2/issues/173

Similar to https://github.com/casnode/casnode-website/pull/93

### CODE REVIEW

File: docs/Middlewares.mdx [hunk 1]
```diff
@@ -45,9 +45,8 @@ authors: [hsluoyz]
...
 |--------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| [Spring Boot](https://projects.spring.io/spring-boot/) | Makes it easy to create Spring-powered applications and services, via plugin: [casbin-spring-boot-starter](https://github.com/jcasbin/casbin-spring-boot-starter) or [jcasbin-springboot-plugin](https://github.com/jcasbin/jcasbin-springboot-plugin) or [Simple SpringBoot security demo with jCasbin](https://github.com/jveverka/spring-examples/tree/master/spring-jcasbin) |
```
Comment (Selflocking) [inline docs/Middlewares.mdx:48]: `jcasbin-springboot-plugin` not found

File: docs/Middlewares.mdx [hunk 2]
```diff
@@ -131,7 +130,7 @@ authors: [hsluoyz]
...
 | [Actix web](https://github.com/actix/actix-web)   | A small, pragmatic, and extremely fast rust web framework, via plugin: [actix-casbin-auth](https://github.com/casbin-rs/actix-casbin-auth)                                                                                                                                                                                                                        |
-| [Rocket](https://github.com/SergioBenitez/Rocket) | a web framework for Rust that makes it simple to write fast, secure web applications without sacrificing flexibility, usability, or type safety, via plugin: [rocket-authz](https://github.com/casbin-rs/rocket-authz) or [rocket-casbin-auth](https://github.com/oiatz/rocket-casbin-auth) or [rocket-casbin-demo](https://github.com/simoin/Rocket_Casbin_Demo) |
```
Comment (Selflocking) [inline docs/Middlewares.mdx:134]: the author of `rocket-casbin-demo` seems to have deleted his account:  https://github.com/casbin/casbin-rs/issues/93#issuecomment-680218729

File: docs/Tutorial.mdx [hunk 3]
```diff
@@ -103,7 +103,6 @@ This is another longer-version paper published in Journal of Software. The citat
 ### Iris
 
 - [Iris + Casbin: Practice for permission management (in Chinese)](https://zxc0328.github.io/2018/05/14/casbin-iris/)
-- [Role-based access control for HTTP based on Casbin (in Chinese)](https://studyiris.com/example/exper/casbin.html)
```
Comment (Selflocking) [inline docs/Tutorial.mdx:106]: not found

File: docs/Tutorial.mdx [hunk 4]
```diff
@@ -161,10 +160,6 @@ This is another longer-version paper published in Journal of Software. The citat
 
 - [Casbin, Powerful and Efficient ACL for Your Projects](http://phpmagazine.net/2018/11/casbin-powerful-and-efficient-acl-for-your-projects.html)
 
-### Laravel
-
-- [Laravel authorization: authorization library supporting ACL, RBAC, ABAC and other models](https://developpaper.com/laravel-authorization-authorization-library-supporting-acl-rbac-abac-and-other-models/)
```
Comment (Selflocking) [inline docs/Tutorial.mdx:166]: not found

File: docs/Middlewares.mdx (hunk 1 above)
Comment (hsluoyz) [reply docs/Middlewares.mdx:48]: Delete it

File: docs/Adapters.mdx [hunk 5]
```diff
@@ -158,7 +158,6 @@ A complete list of Casbin adapters is provided as below. Any 3rd-party contribut
...
 | [EF Adapter](https://github.com/casbin-net/EF-Adapter)                            | ORM   | Casbin                                                 | ❌        | MySQL, PostgreSQL, SQLite, Microsoft SQL Server, Oracle, DB2, etc. are supported by [Entity Framework 6](https://docs.microsoft.com/en-us/ef/ef6/)                                                                                  |
 | [EFCore Adapter](https://github.com/casbin-net/EFCore-Adapter)                    | ORM   | Casbin                                                 | ✅        | MySQL, PostgreSQL, SQLite, Microsoft SQL Server, Oracle, DB2, etc. are supported by [Entity Framework Core](https://docs.microsoft.com/en-us/ef/core/)                                                                              |
-| [EFCore Adapter (.NET Core 5)](https://github.com/g4dvali/CasbinRBACAdapter)      | ORM   | [@g4dvali](https://github.com/g4dvali)                 | ✅        | MySQL, PostgreSQL, SQLite, Microsoft SQL Server, Oracle, DB2, etc. are supported by [Entity Framework Core](https://docs.microsoft.com/en-us/ef/core/)                                                                              |
```
[the remark this answers is not in the document]
Comment (hsluoyz) [reply docs/Adapters.mdx:161]: Delete it

File: docs/Middlewares.mdx [hunk 6]
```diff
@@ -45,9 +45,8 @@ authors: [hsluoyz]
...
 | [Apache Shiro](https://shiro.apache.org/)              | A powerful and easy-to-use Java security framework that performs authentication, authorization, cryptography, and session management, via plugin: [shiro-casbin](https://github.com/jcasbin/shiro-casbin) or [shiro-jcasbin-spring-boot-starter](https://github.com/mapleafgo/shiro-jcasbin-spring-boot-starter)                                                                 |
-| [Vert.x](https://vertx.io/)                            | A tool-kit for building reactive applications on the JVM, via plugin: [vertx-auth-jcasbin](https://github.com/vert-x3/vertx-auth/pull/215)                                                                                                                                                                                                                                       |
```
[the remark this answers is not in the document]
Comment (hsluoyz) [reply docs/Middlewares.mdx:50]: Delete it

File: docs/Middlewares.mdx (hunk 2 above)
Comment (hsluoyz) [reply docs/Middlewares.mdx:134]: Delete them

File: docs/Tutorial.mdx (hunk 3 above)
Comment (hsluoyz) [reply docs/Tutorial.mdx:106]: Delete it

File: docs/Tutorial.mdx (hunk 4 above)
Comment (hsluoyz) [reply docs/Tutorial.mdx:166]: Delete it

File: .github/workflows/docs.links.check.config.json [hunk 7]
```diff
@@ -0,0 +1,13 @@
+{
+    "ignorePatterns": [
+        {
+            "pattern": "^[^h]"
```
Comment (dirname) [inline .github/workflows/docs.links.check.config.json:4]: Same as: https://github.com/casnode/casnode-website/pull/93#discussion_r1257318253

**Review by dirname — COMMENTED** [review]
Some markdown table formatting has been disrupted, perhaps the markdown file needs to be reformatted to improve readability.

**Pull request merged**